### PR TITLE
Consistent ordering of function checks

### PR DESCRIPTION
**Pull request by welteki**
## Description

Print out the namespaces and functions checks in a consistent order.

Additionally changes functions to be named according convention:
`function.namespace` instead of `namespace.function`

## Motivation and Context

- [ ] I have raised an issue to propose this change ([required](https://github.com/openfaas/faas/blob/master/CONTRIBUTING.md))
- [ ] My issue has received approval from the maintainers or lead with the `design/approved` label

https://github.com/openfaas/config-checker/pull/13#issuecomment-1239071864

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I've read the [CONTRIBUTION](https://github.com/openfaas/faas/blob/master/CONTRIBUTING.md) guide
- [x] I have signed-off my commits with `git commit -s`
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -454,8 +454,9 @@ Features detected:
 
 	fmt.Printf("\n")
```
Comment (alexellis) [inline main.go:458]: You should probably short this list of strings (functionNamespaces)

https://gobyexample.com/sorting

File: main.go [hunk 2]
```diff
@@ -517,7 +518,9 @@ Features detected:
 		fmt.Printf("⚠️ Pro gateway detected, but autoscaler is not enabled\n")
 	}
 
-	for namespace, functions := range nsFunctions {
+	for _, namespace := range functionNamespaces {
+		functions := nsFunctions[namespace]
```
Comment (alexellis) [inline main.go]: Best check if this was "OK" for if the namespace is empty

(I forget the order of ok/value), but: 
if ok, value := map[key] ; ok {

}

**Review by alexellis — CHANGES_REQUESTED** [review]
Please see notes.

File: main.go (hunk 1 above)
Comment (welteki) [reply main.go:458]: The namespaces are now sorted.

File: main.go (hunk 2 above)
Comment (welteki) [reply main.go]: I added this check.

**Review by alexellis — APPROVED** [review]
Approved

**Pull request merged**